templates/CSI/rbd/storageclass.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: {{ rbd_storageclass_name | default('csi-rbd') }}
+provisioner: rbd.csi.ceph.com
+parameters:
+    # Comma separated list of Ceph monitors
+    # if using FQDN, make sure csi plugin's dns policy is appropriate.
+    # monitors: mon1:port,mon2:port,...
+    # e.g. 'rook-ceph-mon-a.openshift-storage.svc.cluster.local:6789,
+    # rook-ceph-mon-b.openshift-storage.svc.cluster.local:6789,
+    # rook-ceph-mon-c.openshift-storage.svc.cluster.local:6789'
+    monitors: {{ ceph_monitors | default
+                ('rook-ceph-mon-a.openshift-storage.svc.cluster.local:6789,
+                rook-ceph-mon-b.openshift-storage.svc.cluster.local:6789,
+                rook-ceph-mon-c.openshift-storage.svc.cluster.local:6789') }}
+
+ # if "monitors" parameter is not set, driver to get monitors from same
+    # secret as admin/user credentials. "monValueFromSecret" provides the
+    # key in the secret whose value is the mons
+    #monValueFromSecret: "monitors"
+
+    # Ceph pool into which the RBD image shall be created
+    pool: {{ rbd_pool | default('rbd') }}
+
+    # RBD image format. Defaults to "2".
+    imageFormat: "2"
+
+    # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
+    imageFeatures: layering
+
+    # The secrets have to contain Ceph admin credentials.
+    csi.storage.k8s.io/provisioner-secret-name: {{ csi_rbd_secret | default('csi-rbd-secret') }}
+    csi.storage.k8s.io/provisioner-secret-namespace: {{ csi_rbd_secret_ns | default('default') }}
+    csi.storage.k8s.io/node-publish-secret-name: {{ csi_rbd_secret | default('csi-rbd-secret') }}
+    csi.storage.k8s.io/node-publish-secret-namespace: {{ csi_rbd_secret_ns | default('default') }}
+
+    # Ceph users for operating RBD
+    adminid: admin
+    userid: {{ userid | default('kubernetes') }}
```

Comment:
Can we comment it if we not we need to create  kubernetes user else pvc will remain in Pending

Reply:
same :)

Reply:
I tried running using "userdid" but was failing with some secret-key error. When I commented it and ran it was successful can anyone cross-check this?

Reply:
@nehaberry @prsurve it works with kubernetes user, pvc will be in bound state, last time I made some mistake in code.